client/Pages/EventTypeDetails/QueryTab.elm

```diff
@@ -0,0 +1,231 @@
+module Pages.EventTypeDetails.QueryTab exposing (..)
+
+import Pages.EventTypeDetails.Models exposing (Model)
+import Pages.EventTypeDetails.Messages exposing (..)
+import Stores.Query exposing (Query, queryDecoder)
+import RemoteData
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (..)
+import Helpers.UI exposing (..)
+import Helpers.Panel exposing (renderError, warningMessage)
+import Helpers.Store exposing (errorToViewRecord)
+import RemoteData exposing (WebData, isLoading)
+import Http
+import Json.Decode
+import Config
+import User.Models exposing (Settings)
+import String.Extra exposing (replace)
+import Helpers.Ace as Ace
+
+
+queryTab : Settings -> Model -> Html Msg
+queryTab setting pageState =
+    div [ class "dc-card" ]
+        [ showRemoteDataStatus
+            pageState.loadQueryResponse
+            (queryTabHeader setting pageState)
+        ]
+
+
+queryTabHeader : Settings -> Model -> Query -> Html Msg
+queryTabHeader settings model query =
+    let
+        statClass =
+            case query.status of
+                "active" ->
+                    "schema-tab__value dc-status dc-status--active"
+
+                "inactive" ->
+                    "schema-tab__value dc-status dc-status--error"
+
+                _ ->
+                    "schema-tab__value dc-status dc-status--inactive"
+
+        terminate =
+            if query.status == "active" then
+                span
+                    [ onClick OpenDeleteQueryPopup
+                    , class "icon-link dc-icon--trash dc-btn--destroy dc-icon dc-icon--interactive"
+                    , title "Terminate Query"
```

Comment:
I choose this word because there is no opposite word like "unterminate" in opposite to "Activate/Reactivate" and "Stop". To suggest people that they cannot restart it later.

Reply:
That's a good point